packages/go-ui-storybook/src/stories/ReducedListDisplay.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import { ReducedListDisplayProps } from '@ifrc-go/ui';
+import type {
+    Meta,
+    StoryObj,
+} from '@storybook/react';
+
+import ReducedListDisplay from './ReducedListDisplay';
+
+interface Option {
+    label: string;
+}
+
+const options: Option[] = [
+    { label: 'Green' },
+    { label: 'Yellow' },
+    { label: 'Blue' },
+    { label: 'White' },
+    { label: 'Black' },
+];
+type RendererProps = {
+    children: React.ReactNode
+
+};
```

Comment:
```suggestion
];

type RendererProps = {
    children: React.ReactNode
};
```
Minor thing but make it consistant